Lib/fontTools/ttLib/reorderGlyphs.py

```diff
@@ -258,6 +258,12 @@ def reorderGlyphs(font: ttLib.TTFont, new_glyph_order: List[str]):
             f"* only in old: {set(old_glyph_order) - set(new_glyph_order)}"
         )
 
+    # Glyph 0 must be assigned to a .notdef glyph.
+    # https://learn.microsoft.com/en-us/typography/opentype/spec/recom#glyph-0-the-notdef-glyph
+    if ".notdef" in new_glyph_order:
+        new_glyph_order.remove(".notdef")
+        new_glyph_order.insert(0, ".notdef")
+
```

Comment:
I don't think this is necessary. I'm not arguing that one should not place .notdef as first glyph (I know it's required  and things fail otherwise). But this is a relatively low level API, it should just do as told IMO. E.g. one might imagine this been used for testing purposes. Client can do the right thing as needed.

Reply:
Agree. I removed those lines.